pydoctor/epydoc2stan.py

```diff
@@ -398,6 +400,12 @@ def resolve_types(self):
         for pd in self.parameter_descs:
             if pd.name in self.types:
                 pd.type = self.types[pd.name]
+        if 'return' not in self.types:
+             return
+        if not self.return_desc:
+            self.return_desc = FieldDesc()
```

Comment:
Would it be easier to always have a `return_desc` even when it contains no info, or like it is now, where `return_desc` can be `None`? I'm not sure; it probably also depends on what the code looks like after addressing the other issues.

Reply:
I cleaned this up by moving the return-special-casing to the named constructor and going through the usual "handle" flow.